src/main/java/steve/ui/Messages.java

```diff
@@ -0,0 +1,35 @@
+package steve.ui;
+
+// Prints messages of chatbot
+public class Messages {
+    public static void greeting() {
+        System.out.println("____________________________________________________________\n" +
+                "     Hello! I'm Steve\n" +
+                "     What can I do for you?\n" +
+                "____________________________________________________________");
+    }
+
+    public static void goodbye() {
+        System.out.println("____________________________________________________________\n" +
+                "     \n" + "Bye! Hope to see you again soon =)" + "\n" +
+                "____________________________________________________________");
+    }
+
+    public static void unknown() {
+        String first = "Sorry I don't quite understand!";
+        String second = " Please add a task by entering: \n 1. todo \n 2. deadline \n 3. event \n 4. find \n";
+        String third = "Followed by a description of your task! =)";
+        System.out.println(first + second + third);
```

Comment:
Perhaps you can condense this into a single multi line string. Likewise for several other cases.
```suggestion
        System.out.println("""
        Sorry I don't quite understand! Please add a task by entering: 
        1. todo
        2. deadline
        3. event
        4. find
        Followed by a description of your task! =)""");
```

Reply:
Great suggestion will adopt it